pages.zh/linux/lvm.md

```diff
@@ -0,0 +1,26 @@
+# lvs
+
+> 显示逻辑卷信息。
+> 另见: `lvm`。
+> 更多信息参考:<https://man7.org/linux/man-pages/man8/lvs.8.html>。
+
+- 显示逻辑卷信息
+`lvs`
+
+- 显示所有逻辑卷：
+`lvs -a`
```

Comment:
You need to add an empty line between the example description and the example itself.
```suggestion
- 显示所有逻辑卷：

`lvs -a`
```